quickstart/README.md

```diff
@@ -0,0 +1,63 @@
+# Dagger Quickstart
```

Comment:
can we move this at https://github.com/odpf/dagger/tree/doc/add-quickstart-documentation/docs/docs/guides?

Reply:
Moved.
Have also linked the quickstart in the main README